ospfd/ospf_sr.c

```diff
@@ -1764,11 +1764,11 @@ static int ospf_sr_enabled(struct vty *vty)
 
 DEFUN (sr_sid_label_range,
        sr_sid_label_range_cmd,
-       "segment-routing global-block (0-1048575) (0-1048575)",
+       "segment-routing global-block (8000-1048575) (13000-1048575)",
```

Comment:
LGTM, but where did this `13000` come from?

Reply:
13000 = 8000 + 5000 ;-)

More seriously, 8000 is the Minimum lower bound value for the SRGB as defined in lib/mpls.h and 5000 is the minimum SRGB size also defined in lib/mpls.h. So, the minimum upper bound value for the SRGB is 13000 in order to respect the minimum size value checked in the code.

Reply:
That makes sense, thanks for the explanation :)

Reply:
Maybe this comment should be added to the source? Makes it clear where this number came from if someone looks at the code 75 years from now :)